src/applications/personalization/preferences/sass/preferences.scss

```diff
@@ -0,0 +1,44 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.preferences-row {
```

Comment:
Minor point but it might be nice to call this `.preferences-grid` instead. I had assumed this was going to be a single row and not the entire container for the preference items that would handle the wrapping.

Reply:
Good call, updated.